tests/unit/test_api.py

```diff
@@ -0,0 +1,33 @@
+
+from unittest import TestCase
+
+import mock
+from group_project_v2.json_requests import GET
+
+from group_project_v2.project_api import api_implementation
+from tests.utils import TestWithPatchesMixin
+
+
+class BaseApiTest(TestCase, TestWithPatchesMixin):
```

Comment:
@jbzdak nit: `BaseApiTest`? I believe `TypedProjectAPITests`  would be a better name.